examples/CMSapp/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: 2021 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+if(NOT TARGET G4HepEm::g4HepEm)
+  message(STATUS "Disabling CMSapp (needs G4HepEm)")
+  return()
+endif()
+
+if(Geant4_FOUND)
+  if(NOT Geant4_gdml_FOUND)
+    message(STATUS "Disabling CMSapp (needs Geant4 with GDML support)")
+    return()
+  endif()
+else()
+  message(STATUS "Disabling CMSapp (needs Geant4)")
+  return()
+endif()
+
+# Also need VecGeom 1.1.17 or newer
+if(VecGeom_VERSION VERSION_LESS 1.1.17)
+  message(STATUS "Disabling CMSapp (requires VecGeom 1.1.17 or newer to support full GDML parsing)")
+  return()
+else()
+  if(NOT TARGET VecGeom::vgdml)
+    message(STATUS "Disabling CMSapp (requires VecGeom with GDML support)")
+    return()
+  endif()
+endif()
+
+# CMSapp is the AdePT application using Geant4 to initialize G4HepEm data structures for the CMS geometry
```

Comment:
I think this application does more than just "initialize G4HepEm data structures", right?